tests/cephfs/cephfs_nfs/2_node_nfs.py

```diff
@@ -130,6 +117,8 @@ def run(ceph_cluster, **kw):
         rc = fs_util.cephfs_nfs_mount(
             client1, nfs2, nfs_export_name, nfs_mounting_dir_2
         )
+        for command in commands:
```

Comment:
for loop needs to be before cephfs_nfs_mount for nfs2

Reply:
updated